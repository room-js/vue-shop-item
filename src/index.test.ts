import { mount } from '@vue/test-utils';
import MyComponent from './index.vue';

test('renders correctly', () => {
  const wrapper = mount(MyComponent, {
    slots: {
      default: 'Hey, it is a child component content'
    }
  });
  expect(wrapper).toMatchSnapshot();
});
