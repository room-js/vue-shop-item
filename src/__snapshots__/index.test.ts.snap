// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `<div>Hey, it is a child component content</div>`;
